libazureinit/src/config.rs

```diff
@@ -223,15 +223,23 @@ impl Default for Wireserver {
 ///
 /// Configures telemetry behavior, including diagnostic settings.
 #[derive(Serialize, Deserialize, Debug, Clone)]
+#[serde(default)]
 pub struct Telemetry {
     /// Flag to enable or disable KVP diagnostics. Defaults to `true`.
     pub kvp_diagnostics: bool,
+
+    /// Optional filter directives for the KVP tracing layer. When set,
+    /// these directives are parsed using `tracing_subscriber::EnvFilter` and
+    /// applied to the KVP layer unless overridden by the `AZURE_INIT_KVP_FILTER`
+    /// environment variable. When not set, defaults tailored for azure-init are used.
+    pub kvp_filter: Option<String>,
```

Comment:
provide examples of acceptable values (can use similar examples as the ones found in kvp.rs)